flake.nix

```diff
@@ -76,6 +76,7 @@
       lenovo-legion-16ach6h-hybrid = import ./lenovo/legion/16ach6h/hybrid;
       lenovo-legion-16ach6h-nvidia = import ./lenovo/legion/16ach6h/nvidia;
       lenovo-legion-16ithg6 = import ./lenovo/legion/16ithg6;
+      lenovo-legion-16irx8h = import ./lenovo/legion/16irx8h;
```

Comment:
Please also add it to the markdown table in README.md where our ci script picks it up from.

Reply:
Just learned via https://github.com/johnfanv2/LenovoLegionLinux/issues/81 there is a way to load the lenovo kernel module, will add that and commit together with this.

Reply:
It should be as easy as this:
```nix
  boot.extraModulePackages = [ config.boot.kernelPackages.lenovo-legion-module ];
```

Reply:
@ulrikstrid added the kernel module and fixed the markdown... I'll suggest to add this kernel module to all of Lenovo legions.